closure/goog/base.js

```diff
@@ -1129,7 +1129,8 @@ if (goog.DEPENDENCIES_ENABLED) {
   goog.appendScriptSrcNode_ = function(src) {
     /** @type {Document} */
     var doc = goog.global.document;
-    var scriptEl = doc.createElement('script');
+    var scriptEl = /** @type {HTMLScriptElement} */
```

Comment:
You're probably right, because of {!Element} of Document.prototype.createElement.